docs/DeveloperGuide.md

```diff
@@ -395,7 +433,7 @@ Priorities: High (must have), Medium (nice to have), Low (unlikely to have)
 
 (For all use cases below, the **System** is `TAilor` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case 1 (UC1): Importing an existing database**
+**Use case 1 (UC1): Importing an existing database** *(proposed)*
```

Comment:
Should this be Proposed still? Since the importing from CSV has/will be implemented